### PR TITLE
test: [program-event] add test cases

### PR DESCRIPTION
**Pull request by mmmateos**
## Description
Please include a summary of the change and include the related jira issue if it exists.

[ jira issue ](https://jira.dhis2.org/browse/

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: app/src/androidTest/java/org/dhis2/usescases/programevent/ProgramEventTest.kt [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        val eventOrgUnit = "Ngelehun CHC"
+        prepareProgramAndLaunchActivity()
+
+        programEventsRobot {
+            clickOnAddEvent()
+        }
+
+        eventRegistrationRobot {
+            clickNextButton()
+            waitToDebounce(600)
```
Comment (JaimeToca) [inline app/src/androidTest/java/org/dhis2/usescases/programevent/ProgramEventTest.kt:30]: if you remove this the test fails?

Comment (JaimeToca) [reply app/src/androidTest/java/org/dhis2/usescases/programevent/ProgramEventTest.kt:30]: @mmmateos

File: app/src/androidTest/java/org/dhis2/usescases/programevent/ProgramEventTest.kt [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+            clickOnEvent(eventDate, eventOrgUnit)
+        }
+
+        eventRobot {
+            checkDetails(eventDate, eventOrgUnit)
+        }
+    }
+
+    @Test
+    fun shouldAddNoteOnExistingEvent() {
```
Comment (JaimeToca) [inline app/src/androidTest/java/org/dhis2/usescases/programevent/ProgramEventTest.kt:60]: @ferdyrod do you think this test is needed? Having already notes tests?

File: app/src/androidTest/java/org/dhis2/usescases/programevent/ProgramEventTest.kt (hunk 1 above)
Comment (mmmateos) [reply app/src/androidTest/java/org/dhis2/usescases/programevent/ProgramEventTest.kt:30]: Yes, without it the test fails

File: app/src/androidTest/java/org/dhis2/usescases/programevent/ProgramEventTest.kt (hunk 2 above)
Comment (ferdyrod) [reply app/src/androidTest/java/org/dhis2/usescases/programevent/ProgramEventTest.kt:60]: Notes already has its own test, this is not needed. Unless this was intendend as a flow test.

**Review by ferdyrod — COMMENTED** [review]
.

**Review by ferdyrod — APPROVED** [review]

**Review by JaimeToca — APPROVED** [review]

**Pull request merged**